components/common/Config.brs

```diff
@@ -0,0 +1,5 @@
+sub init()
+  'Todo: load in a task
+  config = parseJson(ReadAsciiFile("pkg:/config/config.json"))
+  m.global.addFields({"config": config})
```

Comment:
why instead of using a config node, we just load the config directly in the scene